packages/ui/README.md

```diff
@@ -68,39 +68,45 @@ npm install -D @esbuild-plugins/node-globals-polyfill @rollup/plugin-inject roll
 Then you have to adjust the **vite.config.js** file to prepare the project for all the Solana packages previously installed.
 
 ```javascript
-import { sveltekit } from '@sveltejs/kit/vite'
+import { sveltekit } from '@sveltejs/kit/vite';
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import inject from '@rollup/plugin-inject';
+import nodePolyfills from "rollup-plugin-node-polyfills";
+import path from 'path';
 
+/** @type {import('vite').UserConfig} */
 const config = {
 	plugins: [sveltekit()],
-	optimizeDeps: {
-		include: ['@solana/web3.js', 'buffer'],
-		esbuildOptions: {
-				target: 'esnext',
-				plugins: [NodeGlobalsPolyfillPlugin({ buffer: true })],
+  optimizeDeps: {
+    include: ['@project-serum/anchor', '@solana/web3.js'],
```

Comment:
@smedegaard thanks for the contribution. What is the reason for theses changes? Thanks!